hoomd/md/methods.py

```diff
@@ -475,21 +499,131 @@ class nph(NPT):
          :math:`W=d N T_0 \tau_P^2`, where :math:`d` is the dimensionality and :math:`N` the number
          of particles.
 
-    :py:class:`nph` is an integration method and must be used with ``mode_standard``.
+    :py:class:`NPH` is an integration method and must be used with ``mode_standard``.
```

Comment:
```suggestion
```
Remove: refers to v2.

Add NPH to the list of supported integrators in `md/integrate.py` ~ line 127.

Reply:
Also, just above this in the file - you can remove the "The available options are identical" sentence as all options are now explicitly documented. For the note regarding tauS - copy the new language used in NPTs documentation.